src/client/dfuse/dfuse_core.c

```diff
@@ -257,32 +256,32 @@ dfuse_start(struct dfuse_info *dfuse_info, struct dfuse_dfs *dfs)
 	args.allocated = 1;
 	args.argv = calloc(sizeof(*args.argv), args.argc);
```

Comment:
why don't these use D_ALLOC ? D_ASPRINTF, D_STRNDUP, etc ?

Reply:
It's a really weird part of the libfuse interface, it needs you to pass in argc/argv which it then modifies in place to add new options internal to libfuse so at the end of it you get an array of pointers, some of which allocated by dfuse and some by libfuse.  These used to use the macros but then we needed to whitelist them as "not really lost" memory, the tidiest solution was just not to use the macros.